packages/login/src/components/passwordUpdate/PasswordUpdateDialog/PasswordMatchDisplay.js

```diff
@@ -3,13 +3,13 @@ import {FormattedMessage} from 'react-intl'
 
 const PasswordMatchDisplay = props => {
   const message = (props.passwordRepeat && props.password !== props.passwordRepeat)
-    ? <FormattedMessage id="client.login.passwordUpdate.noMatch"/>
+    ? <li className="text-danger"><FormattedMessage id="client.login.passwordUpdate.noMatch"/></li>
     : '\u00a0'
 
   return (
-    <div className="PasswordMatchDisplay text-danger">
+    <ul className="PasswordMatchDisplay icon-list">
```

Comment:
Camelcase classnames are valid but risky, because it is not guaranteed by specification that browsers are case-sensitive (password-match-display would be safe). Camelcase was proposed and introduced earlier of your team to match React's classnames. I prefer lowercase and hyphens. I checked the usage and agree that we do not need it anymore.